e2e/support/helpers/e2e-embedding-sdk-helpers.ts

```diff
@@ -15,5 +27,51 @@ export function signInAsAdminAndEnableEmbeddingSdkForE2e() {
   });
 }
 
+export const mockAuthProviderAndJwtSignIn = (user = USERS.admin) => {
```

Comment:
It's moved from `component` test folder, because it contains JSX, that Cypress can't parse during `e2e` testing